caas/mixins.spec

```diff
@@ -32,7 +32,7 @@ ethernet: dhcp
 camera-ext: ext-camera-only
 rfkill: true(force_disable=)
 wlan: iwlwifi(libwifi-hal=true)
-codecs: configurable(sw_omx_video=false, hw_omx_video=false, platform=tgl, profile_file=media_profiles_1080p.xml, gpu=gen12)
+codecs: configurable(enable_msdk_c2=true, sw_omx_video=false, hw_omx_video=false, platform=tgl, profile_file=media_profiles_1080p.xml, gpu=gen12)
```

Comment:
We have already set this variable in codec2. Please don't set it in omx again.

Thanks,
Tianmi